datafusion/tests/sql.rs

```diff
@@ -2395,9 +2394,14 @@ async fn explain_analyze_baseline_metrics() {
     );
     assert_metrics!(
         &formatted,
-        "GlobalLimitExec: limit=1, ",
+        "GlobalLimitExec: limit=3, ",
         "metrics=[output_rows=1, elapsed_compute="
     );
+    assert_metrics!(
+        &formatted,
+        "LocalLimitExec: limit=3",
```

Comment:
by making the query more complicated, it has also introduced a `LocalLimitExec` for testing 🎉